### PR TITLE
MINOR: Streams upgrade system test cleanup

### PR DESCRIPTION
**Pull request by bbejeck**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
This is a follow-up PR to perform the same clean-up from #7552 

Ran all system tests http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2019-10-21--001.1571679740--bbejeck--MINOR_clean_up_streams_upgrade_system_test--8e61da4/report.html 

All tests ran.  Note there is one failure, but it' unrelated to these changes.  I'm currently investigating the failure.  

ping @guozhangwang @mjsax

**Comment by guozhangwang** [comment]
The change lgtm modulo the test failure, ping @mjsax again.

**Comment by bbejeck** [comment]
Turns out the test failure was related to logging changes in #7537 and I just needed to update the expected log statements.  I ran a new branch builder for the failing test

http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2019-10-23--001.1571861801--bbejeck--MINOR_clean_up_streams_upgrade_system_test--e5f5697/report.html

~The test is showing some flakiness, but it's unrelated to the changes in this PR and should be pursued in a separate follow-on effort~.

**Comment by bbejeck** [comment]
ping @guozhangwang and @mjsax updated this

File: streams/upgrade-system-tests-0102/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java [hunk 1]
```diff
@@ -32,17 +32,14 @@
 
     @SuppressWarnings("unchecked")
     public static void main(final String[] args) throws Exception {
-        if (args.length < 2) {
-            System.err.println("StreamsUpgradeTest requires two argument (kafka-url, properties-file) but only " + args.length + " provided: "
-                + (args.length > 0 ? args[0] : ""));
+        if (args.length < 1) {
+            System.err.println("StreamsUpgradeTest requires one argument (properties-file) but  provided none");
```
Comment (abbccdda) [inline streams/upgrade-system-tests-0102/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java:36]: Why do we remove args in print?

File: streams/upgrade-system-tests-10/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java [hunk 2]
```diff
@@ -32,17 +32,14 @@
 
     @SuppressWarnings("unchecked")
     public static void main(final String[] args) throws Exception {
-        if (args.length < 2) {
-            System.err.println("StreamsUpgradeTest requires two argument (kafka-url, properties-file) but only " + args.length + " provided: "
-                + (args.length > 0 ? args[0] : ""));
+        if (args.length < 1) {
```
Comment (abbccdda) [inline streams/upgrade-system-tests-10/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java:35]: Will we hit NPE in this case?

**Comment by bbejeck** [comment]
updated with changes from #7586 by @ableegoldman. Re-ran branch builder with 10 repeats and the version probing test passes now 
http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2019-10-23--001.1571868682--bbejeck--MINOR_clean_up_streams_upgrade_system_test--b626e9a/report.html

**Comment by guozhangwang** [comment]
LGTM! Please feel free to merge @bbejeck

**Comment by bbejeck** [comment]
Merged #7571 into trunk

**Pull request merged**